diff --git a/clientside_validation_jquery/js/cv.jquery.ckeditor.js b/clientside_validation_jquery/js/cv.jquery.ckeditor.js
index 1a77067..a2acd69 100644
--- a/clientside_validation_jquery/js/cv.jquery.ckeditor.js
+++ b/clientside_validation_jquery/js/cv.jquery.ckeditor.js
@@ -30,7 +30,7 @@
           return;
         }
         validator.settings.ignore = ignore;
-        validator.settings.errorPlacement = function(place, $element) {
+        validator.settings.errorPlacement = function (place, $element) {
           var id = $element.attr('id');
           var afterElement = $element[0];
           if (CKEDITOR.instances.hasOwnProperty(id)) {
diff --git a/clientside_validation_jquery/js/cv.jquery.ife.js b/clientside_validation_jquery/js/cv.jquery.ife.js
index 369ead8..471cb7e 100644
--- a/clientside_validation_jquery/js/cv.jquery.ife.js
+++ b/clientside_validation_jquery/js/cv.jquery.ife.js
@@ -7,7 +7,7 @@
   // We do this to display the error markup same as in inline_form_errors.
   $(document).once('cvjquery').on('cv-jquery-validate-options-update', function (event, options) {
     options.errorElement = 'strong';
-    options.showErrors = function(errorMap, errorList) {
+    options.showErrors = function (errorMap, errorList) {
       // First remove all errors.
       for (var i in errorList) {
         $(errorList[i].element).parent().find('.form-item--error-message').remove();
diff --git a/clientside_validation_jquery/js/cv.jquery.validate.js b/clientside_validation_jquery/js/cv.jquery.validate.js
index 3a2907e..1188749 100644
--- a/clientside_validation_jquery/js/cv.jquery.validate.js
+++ b/clientside_validation_jquery/js/cv.jquery.validate.js
@@ -62,7 +62,7 @@
         }
       }
 
-      $(context).find('form').once('cvJqueryValidate').each(function() {
+      $(context).find('form').once('cvJqueryValidate').each(function () {
         $(this).validate(drupalSettings.cvJqueryValidateOptions);
       });
     }
