diff --git a/src/Plugin/Field/FieldFormatter/ColorFieldFormatterSwatch.php b/src/Plugin/Field/FieldFormatter/ColorFieldFormatterSwatch.php
index afec0d3..100ebde 100644
--- a/src/Plugin/Field/FieldFormatter/ColorFieldFormatterSwatch.php
+++ b/src/Plugin/Field/FieldFormatter/ColorFieldFormatterSwatch.php
@@ -54,7 +54,7 @@ class ColorFieldFormatterSwatch extends FormatterBase {
       '#title' => $this->t('Width'),
       '#default_value' => $this->getSetting('width'),
       '#min' => 1,
-      $this->t('Defaults to pixels (px) if a number is entered, otherwise, you can enter any unit (ie %, em, vw)')
+      $this->t('Defaults to pixels (px) if a number is entered, otherwise, you can enter any unit (ie %, em, vw)'),
     ];
     $elements['height'] = [
       '#type' => 'textfield',
