diff --git a/src/Controller/ConfigSplitController.php b/src/Controller/ConfigSplitController.php
index 95e9a13..fbb65d7 100644
--- a/src/Controller/ConfigSplitController.php
+++ b/src/Controller/ConfigSplitController.php
@@ -3,8 +3,6 @@
 namespace Drupal\config_split\Controller;
 
 use Drupal\Core\Controller\ControllerBase;
-use Symfony\Component\HttpFoundation\RedirectResponse;
-use Drupal\Core\Url;
 
 /**
  * Class ConfigSplitController.
