diff --git a/js/geolocation-widget-html5.js b/js/geolocation-widget-html5.js
index fd9ec2e..e9c142b 100644
--- a/js/geolocation-widget-html5.js
+++ b/js/geolocation-widget-html5.js
@@ -84,7 +84,6 @@
 
       // Get the geolocation from the browser.
       navigator.geolocation.getCurrentPosition(
-
         // Success handler for getCurrentPosition()
         function (position) {
           var lat = position.coords.latitude;
diff --git a/modules/geolocation_geometry/geolocation_geometry.views.inc b/modules/geolocation_geometry/geolocation_geometry.views.inc
index 4d3614e..15d167c 100644
--- a/modules/geolocation_geometry/geolocation_geometry.views.inc
+++ b/modules/geolocation_geometry/geolocation_geometry.views.inc
@@ -216,7 +216,7 @@ function geolocation_geometry_field_views_data(FieldStorageConfigInterface $fiel
 /**
  * Implements hook_field_views_data_views_data_alter().
  */
-function geolocation_geometry_field_views_data_views_data_alter(array &$data, \Drupal\field\FieldStorageConfigInterface $field_storage) {
+function geolocation_geometry_field_views_data_views_data_alter(array &$data, FieldStorageConfigInterface $field_storage) {
   $entity_type = $field_storage->getTargetEntityTypeId();
   /** @var \Drupal\Core\Entity\Sql\SqlEntityStorageInterface $entity_storage */
   $entity_storage = \Drupal::entityTypeManager()->getStorage($entity_type);
diff --git a/modules/geolocation_google_maps/js/MapFeature/geolocation-control-geocoder.js b/modules/geolocation_google_maps/js/MapFeature/geolocation-control-geocoder.js
index dd8b002..77accc9 100644
--- a/modules/geolocation_google_maps/js/MapFeature/geolocation-control-geocoder.js
+++ b/modules/geolocation_google_maps/js/MapFeature/geolocation-control-geocoder.js
@@ -27,7 +27,6 @@
          */
         function (map, featureSettings) {
           Drupal.geolocation.geocoder.addResultCallback(
-
             /**
              * @param {Object} address.geometry.bounds
              */
diff --git a/modules/geolocation_yandex/js/MapFeature/control-zoom.js b/modules/geolocation_yandex/js/MapFeature/control-zoom.js
index 87dff0a..1d8f649 100644
--- a/modules/geolocation_yandex/js/MapFeature/control-zoom.js
+++ b/modules/geolocation_yandex/js/MapFeature/control-zoom.js
@@ -34,6 +34,7 @@
               // Leave the default values.
               options = {};
               break;
+
             case "right":
               // Size adaptivity will be disabled.
               options = {
