diff --git a/hotjar.api.php b/hotjar.api.php
index b8921ce..3685540 100644
--- a/hotjar.api.php
+++ b/hotjar.api.php
@@ -1,4 +1,5 @@
 <?php
+
 /**
  * @file
  * Hooks provided by the Hotjar module.
@@ -64,7 +65,7 @@ function hook_hotjar_settings_alter(array &$settings) {
  * attached as an asset.
  *
  * @param string $script
- *    The Hotjar activation script.
+ *   The Hotjar activation script.
  */
 function hook_hotjar_snippet_alter(&$script) {
   $script = 'window.enableHotjar = function() { ' . $script . ' };';
diff --git a/js/hotjar.js b/js/hotjar.js
index bddd69e..d758d93 100644
--- a/js/hotjar.js
+++ b/js/hotjar.js
@@ -9,12 +9,12 @@
     return;
   }
 
-  (function(h,o,t,j,a,r){
-    h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
-    h._hjSettings={hjid:drupalSettings.hotjar.account,hjsv:drupalSettings.hotjar.snippetVersion};
-    a=o.getElementsByTagName('head')[0];
-    r=o.createElement('script');r.async=1;
-    r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
+  (function (h,o,t,j,a,r) {
+    h.hj = h.hj||function () {(h.hj.q = h.hj.q||[]).push(arguments)};
+    h._hjSettings = {hjid:drupalSettings.hotjar.account,hjsv:drupalSettings.hotjar.snippetVersion};
+    a = o.getElementsByTagName('head')[0];
+    r = o.createElement('script');r.async = 1;
+    r.src = t + h._hjSettings.hjid + j + h._hjSettings.hjsv;
     a.appendChild(r);
   })(window, document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
 })(drupalSettings);
