diff --git a/tests/src/Functional/KeyInputTest.php b/tests/src/Functional/KeyInputTest.php
index 1c1cbd2..c9bf222 100644
--- a/tests/src/Functional/KeyInputTest.php
+++ b/tests/src/Functional/KeyInputTest.php
@@ -2,9 +2,7 @@
 
 namespace Drupal\Tests\key\Functional;
 
-use Drupal\Component\Render\FormattableMarkup;
 use Drupal\Tests\BrowserTestBase;
-use Drupal\Core\Url;
 
 /**
  * Tests input of keys.
