diff --git a/src/Form/GrantsForm.php b/src/Form/GrantsForm.php
index 3aca893..9ae13f4 100644
--- a/src/Form/GrantsForm.php
+++ b/src/Form/GrantsForm.php
@@ -109,7 +109,7 @@ class GrantsForm extends FormBase {
       }
       // Calculate default grants for found users.
       if (isset($form_values['uid']) && is_array($form_values['uid'])) {
-        // set the cast type depending on which database engine is being used.
+        // Set the cast type depending on which database engine is being used.
         if (strstr($db->version(), 'MariaDB') !== FALSE) {
           $cast_type = 'int';
         }
@@ -117,7 +117,7 @@ class GrantsForm extends FormBase {
           $cast_type = 'integer';
         }
         else {
-          // assume it's MySQL.
+          // Assume it's MySQL.
           $cast_type = 'unsigned';
         }
         foreach (array_keys($form_values['uid']) as $uid) {
