diff --git a/src/PasswordPolicyValidator.php b/src/PasswordPolicyValidator.php
index a7111da..3ddfad2 100644
--- a/src/PasswordPolicyValidator.php
+++ b/src/PasswordPolicyValidator.php
@@ -6,6 +6,9 @@ use Drupal\Core\Entity\EntityTypeManagerInterface;
 use Drupal\Core\Password\PasswordInterface;
 use Drupal\user\UserInterface;
 
+/**
+ *
+ */
 class PasswordPolicyValidator implements PasswordPolicyValidatorInterface {
 
   /**
