diff --git a/src/Plugin/Derivative/SimpleBlock.php b/src/Plugin/Derivative/SimpleBlock.php
index 71d76bc..1a12ac8 100644
--- a/src/Plugin/Derivative/SimpleBlock.php
+++ b/src/Plugin/Derivative/SimpleBlock.php
@@ -48,7 +48,7 @@ class SimpleBlock extends DeriverBase implements ContainerDeriverInterface {
       $this->derivatives[$simple_block->id()] = $base_plugin_definition;
       $this->derivatives[$simple_block->id()]['admin_label'] = $simple_block->label();
       $this->derivatives[$simple_block->id()]['config_dependencies'][$simple_block->getConfigDependencyKey()] = [
-        $simple_block->getConfigDependencyName()
+        $simple_block->getConfigDependencyName(),
       ];
     }
     return parent::getDerivativeDefinitions($base_plugin_definition);
diff --git a/src/SimpleBlockEditForm.php b/src/SimpleBlockEditForm.php
index 0a88e1a..dde6205 100644
--- a/src/SimpleBlockEditForm.php
+++ b/src/SimpleBlockEditForm.php
@@ -49,14 +49,14 @@ class SimpleBlockEditForm extends EntityForm implements ContainerInjectionInterf
       '#description' => $this->t("The block title."),
       '#required' => TRUE,
     ];
-    $form['content'] = array(
+    $form['content'] = [
       '#type' => 'text_format',
       '#format' => $simple_block->getContent()['format'],
       '#title' => $this->t('Block content'),
       '#default_value' => $simple_block->getContent()['value'],
       '#description' => $this->t("The block content."),
       '#required' => TRUE,
-    );
+    ];
     return $form;
   }
 
diff --git a/src/SimpleBlockListBuilder.php b/src/SimpleBlockListBuilder.php
index 2ee4981..769a708 100644
--- a/src/SimpleBlockListBuilder.php
+++ b/src/SimpleBlockListBuilder.php
@@ -27,10 +27,10 @@ class SimpleBlockListBuilder extends ConfigEntityListBuilder {
    * {@inheritdoc}
    */
   public function buildRow(EntityInterface $entity) {
-      return [
-        'id' => $entity->id(),
-        'block' => $entity->label(),
-      ] + parent::buildRow($entity);
+    return [
+      'id' => $entity->id(),
+      'block' => $entity->label(),
+    ] + parent::buildRow($entity);
   }
 
   /**
