diff --git a/src/Element/TimeElement.php b/src/Element/TimeElement.php
index a19b61c..7e89130 100644
--- a/src/Element/TimeElement.php
+++ b/src/Element/TimeElement.php
@@ -30,7 +30,7 @@ class TimeElement extends FormElement {
   public function getInfo() {
     $class = get_class($this);
     return [
-      '#show_seconds' => false,
+      '#show_seconds' => FALSE,
       '#input' => TRUE,
       '#process' => [
         [$class, 'processAjaxForm'],
diff --git a/src/Plugin/Field/FieldWidget/TimeRangeWidget.php b/src/Plugin/Field/FieldWidget/TimeRangeWidget.php
index a77f124..aa25190 100644
--- a/src/Plugin/Field/FieldWidget/TimeRangeWidget.php
+++ b/src/Plugin/Field/FieldWidget/TimeRangeWidget.php
@@ -50,7 +50,6 @@ class TimeRangeWidget extends WidgetBase {
       ];
     }
 
-
     $show_seconds = (bool) $this->getSetting('enabled');
     if ($show_seconds) {
       $element['from']['#attributes']['step'] = $this->getSetting('step');
@@ -66,12 +65,11 @@ class TimeRangeWidget extends WidgetBase {
   /**
    * {@inheritdoc}
    */
-  public static function defaultSettings()
-  {
+  public static function defaultSettings() {
     return [
-        'enabled' => FALSE,
-        'step' => 5,
-      ] + parent::defaultSettings();
+      'enabled' => FALSE,
+      'step' => 5,
+    ] + parent::defaultSettings();
   }
 
   /**
@@ -79,22 +77,23 @@ class TimeRangeWidget extends WidgetBase {
    */
   public function settingsForm(array $form, FormStateInterface $form_state) {
     return [
-        'enabled' => [
-          '#type' => 'checkbox',
-          '#title' => $this->t('Add seconds parameter to input widget'),
-          '#default_value' => $this->getSetting('enabled'),
-        ],
-        'step' => [
-          '#type' => 'textfield',
-          '#title' => $this->t('Step to change seconds'),
-          '#open' => TRUE,
-          '#default_value' => $this->getSetting('step'),
-          '#states' => [
-            'visible' => [
-              ':input[name$="[enabled]"]' => ['checked' => TRUE],
-            ],
+      'enabled' => [
+        '#type' => 'checkbox',
+        '#title' => $this->t('Add seconds parameter to input widget'),
+        '#default_value' => $this->getSetting('enabled'),
+      ],
+      'step' => [
+        '#type' => 'textfield',
+        '#title' => $this->t('Step to change seconds'),
+        '#open' => TRUE,
+        '#default_value' => $this->getSetting('step'),
+        '#states' => [
+          'visible' => [
+            ':input[name$="[enabled]"]' => ['checked' => TRUE],
           ],
         ],
-      ] + parent::settingsForm($form, $form_state);
+      ],
+    ] + parent::settingsForm($form, $form_state);
   }
+
 }
