diff --git a/tests/src/Unit/FieldValue/FieldRawValueTest.php b/tests/src/Unit/FieldValue/FieldRawValueTest.php
index 42a45fb..8869c75 100644
--- a/tests/src/Unit/FieldValue/FieldRawValueTest.php
+++ b/tests/src/Unit/FieldValue/FieldRawValueTest.php
@@ -34,7 +34,7 @@ class FieldRawValueTest extends UnitTestCase {
    */
   protected function mockFieldItem($values) {
 
-    $field_item  = $this->getMockBuilder('Drupal\Core\Field\FieldItemBase')
+    $field_item = $this->getMockBuilder('Drupal\Core\Field\FieldItemBase')
       ->disableOriginalConstructor()
       ->getMock();
     $field_item->expects($this->any())
diff --git a/tests/src/Unit/FieldValue/FieldTargetEntityTest.php b/tests/src/Unit/FieldValue/FieldTargetEntityTest.php
index f25e0b1..e3599b6 100644
--- a/tests/src/Unit/FieldValue/FieldTargetEntityTest.php
+++ b/tests/src/Unit/FieldValue/FieldTargetEntityTest.php
@@ -18,6 +18,9 @@ class FieldTargetEntityTest extends UnitTestCase {
    */
   protected $extension;
 
+  /**
+   *
+   */
   protected function setUp() {
     $this->extension = new FieldValueExtension();
   }
