diff --git a/src/Plugin/views/filter/NodeStatus.php b/src/Plugin/views/filter/NodeStatus.php
index 8fe9c81..3d4c7fc 100644
--- a/src/Plugin/views/filter/NodeStatus.php
+++ b/src/Plugin/views/filter/NodeStatus.php
@@ -4,7 +4,6 @@ namespace Drupal\view_unpublished\Plugin\views\filter;
 
 use Drupal\node\Entity\NodeType;
 use Drupal\node\Plugin\views\filter\Status;
-use Drupal\views\Annotation\ViewsFilter;
 
 /**
  * Filter by view all unpublished permissions granted by view_unpublished.
